jni/src/main/java/com/horizen/certnative/NaiveThresholdSignatureWKeyRotation.java

```diff
@@ -16,10 +16,28 @@ public class NaiveThresholdSignatureWKeyRotation {
         Library.load();
     }
 
+    private static native FieldElement nativeCreateMsgToSign(
+        BackwardTransfer[] bt,
+        FieldElement scId,
+        int epochNumber,
+        FieldElement endCumulativeScTxCommTreeRoot,
+        long btrFee,
+        long ftMinAmount,
+        FieldElement[] customFields
+    );
+
     public static FieldElement createMsgToSign(
         WithdrawalCertificate withdrawalCertificate
     ) {
-        return withdrawalCertificate.getHash();
+        return nativeCreateMsgToSign(
```

Comment:
As in Rust side: let's call NaiveThresholdSignature.createMsgToSign function as the implementation should be exactly the same

Reply:
Done